### ISSUE TITLE
Article metadata

### ISSUE THREAD

**Issue by edwardabraham** [scholmd/scholmd#10]
I have created a wiki page with a preliminary proposal for article metadata: https://github.com/scholmd/scholmd/wiki/Article-metadata.  

The idea is to follow [Martin's blogpost](http://blog.martinfenner.org/2013/06/29/metadata-in-scholarly-markdown/), basing the metadata fields on https://github.com/citation-style-language/schema/blob/master/csl-data.json

I would appreciate comments on affiliation being the id of an organization element, on the abstract specifying a section (rather than including all the abstract in the YAML), and on anything else that you feel should be changed or added.

Either edit the wiki page, or respond to this issue.

**Comment by mfenner** [comment]
I agree that the abstract shouldn't go in the YAML header, that is really a hack. The evolving standard for institutional identifiers is ISNI/RInggold, for example used by ORCID.
